datasync/chaindatafetcher/kafka/config.go

```diff
@@ -51,6 +52,7 @@ type KafkaConfig struct {
 	Partitions           int32 // Partitions is the number of partitions of a topic.
 	Replicas             int16 // Replicas is a replication factor of kafka settings. This is the number of the replicated partitions in the kafka cluster.
 	SegmentSizeBytes     int   // SegmentSizeBytes is the size of kafka message segment
+	ConsumerBufferSize   int   // ConsumerBufferSize is the maximum size of consumer buffer
```

Comment:
For make it sure, is this limits the number of items in the buffer, not the actual buffer size?

Reply:
You're right. I renamed it. 2178a56